app/models/user.rb

```diff
@@ -0,0 +1,42 @@
+class User < ApplicationRecord
+  has_secure_password
+  # has_secure_password cung cấp: # rubocop:disable Style/AsciiComments
+  # - Các thuộc tính ảo: password, password_confirmation # rubocop:disable Style/AsciiComments
+  # - Trường password_digest để lưu hash z# rubocop:disable Style/AsciiComments
+  # - Phương thức authenticate(password) để xác thực # rubocop:disable Style/AsciiComments
+
+  enum gender: {male: 0, female: 1, other: 2}
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  NAME_MAX_LENGTH = 50
+  EMAIL_MAX_LENGTH = 255
+  MAX_YEARS_AGO = 100
+
+  before_save :downcase_email
+
+  scope :recent, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :recent, -> {order(created_at: :desc)}
```
Thêm space trước block {} e nha